src/codegen/updating-llvm.md

```diff
@@ -70,27 +75,18 @@ through each in detail.
 1. Create a new branch in the rust-lang/llvm-project repository. This branch
    should be named `rustc/a.b-yyyy-mm-dd` where `a.b` is the current version
    number of LLVM in-tree at the time of the branch and the remaining part is
-   today's date.
+   today's date. Move this branch to the commit in LLVM that you'd like, which
+   for this is probably the current LLVM HEAD.
 
 2. Apply Rust-specific patches to the llvm-project repository. All features and
    bugfixes are upstream, but there's often some weird build-related patches
    that don't make sense to upstream which we have on our repositories. These
    patches are around the latest patches in the rust-lang/llvm-project branch
    that rustc is currently using.
 
-3. Update the `compiler-rt` submodule in the
-   `rust-lang-nursery/compiler-builtins` repository. Push this update to the
-   same branch name of the `llvm-project` submodule to the
-   of the `rust-lang/compiler-rt` repository. Then push this update to a branch
-   of `compiler-builtins` with the same-named branch. Note that this step is
-   frequently optional since we may not need to update `compiler-rt`.
-
-4. Prepare a commit to rust-lang/rust
-
-  * Update `src/llvm-project`
-  * Update `compiler-builtins` crate in `Cargo.lock` (if necessary)
-
-5. Build your commit. Make sure you've committed the previous changes to ensure
+3. Build the new LLVM in the `rust` repository. To do this you'll want to update
+   the `src/llvm-project` repository to your branch and the revision you've
+   created. Make sure you've committed changes to `src/llvm-project` to ensure
```

Comment:
We should make sure to update the branch name in `.gitmodules` too, as this enables easy `git submodules update --remote src/llvm-project`.